source/PyMaterialX/PyMaterialXRenderOsl/PyOslRenderer.cpp

```diff
@@ -19,6 +19,7 @@ void bindPyOslRenderer(py::module& mod)
         .def("createProgram", static_cast<void (mx::OslRenderer::*)(const mx::ShaderPtr)>(&mx::OslRenderer::createProgram))
         .def("createProgram", static_cast<void (mx::OslRenderer::*)(const mx::OslRenderer::StageMap&)>(&mx::OslRenderer::createProgram))
         .def("validateInputs", &mx::OslRenderer::validateInputs)
+        .def("updateUniform", &mx::OslRenderer::updateUniform)
```

Comment:
Perhaps this OSL-specific Python binding is not needed, as `OslRenderer` will inherit the default behavior from `ShaderRenderer`?